vmsg.js

```diff
@@ -137,13 +139,153 @@ function inlineWorker() {
   };
 }
 
-class Form {
-  constructor(opts = {}, resolve, reject) {
+class Recorder {
+  constructor(opts = {}, onStop) {
     // Can't use relative URL in blob worker, see:
     // https://stackoverflow.com/a/22582695
     this.wasmURL = new URL(opts.wasmURL || "/static/js/vmsg.wasm", location).href;
     this.shimURL = new URL(opts.shimURL || "/static/js/wasm-polyfill.js", location).href;
+    this.onStop = onStop;
     this.pitch = opts.pitch || 0;
+    this.audioCtx = null;
+    this.gainNode = null;
+    this.pitchFX = null;
+    this.encNode = null;
+    this.worker = null;
+    this.workerURL = null;
+    this.blob = null;
+    this.blobURL = null;
+    this._resolve = null;
```

Comment:
Better to use variables without underscore for consistency. (Same for next line.)

Reply:
fixed :)